extra-games/fortunes-zh/autobuild/patches/fix-fortunes-zh-path.patch

```diff
@@ -0,0 +1,11 @@
+--- a/fortune-zh	2021-11-03 15:18:50.083146802 +0800
```

Comment:
Please add serial number to patch's name to comply our [Styling Manual about patch naming for other source](https://wiki.aosc.io/developer/packaging/package-styling-manual/#other-sources).

Reply:
@eatradish Ping?